### DISCUSSION TITLE
Any way this could work on Intel-based models?

### DISCUSSION THREAD

**Discussion by cwigd** [DavidS95/Smokeless_UMAF#6] [Q&A] [answered]
Hi there,

I suspect I know the answer to this... but does this tool work on Intel-based machines? I just gave it a go on my Lenovo Yoga 7i (14" Intel Evo version). I was able to boot the main menu, but the "Device Manager" screen doesn't show any devices. Makes sense, I suppose... unless I'm missing something.

Alternatively, any idea if a similar method exists to access the expanded BIOS options on an Intel-based Lenovo? It's my only hope for ditching the horrid "Modern Standby" function in favor of S3.

Thanks in advance!

**Comment by solaris111** [comment]
You have ask exactly what I want :)

**Comment by leonardcoutinho** [comment]
I have an Ryzen 5 2500u and don't show anything too.

**Comment by DavidS95** [comment, marked as answer]
It is entirely based on how the bios is configured. The reason its mentioned as an amd tool is because the vast majority of bioses set up for amd systems are compatible with how this tool accesses these hidden settings. If it does not show up that is primarily because of how the bios is made and there is nothing I am able to do about it. This tool can work on some intel systems when the bioses are set up how this tool needs them to be however this is a far rarer occurrence on intel laptops. If the setting do not show up there is nothing I am able to do in order to make it work. Sorry for the bad news. If I had to guess the success rate for amd laptops is over 95% while for intel its probably closer to the opposite at sub 25%.

**Comment by Espionage724** [comment]
`UMAF_BETA.zip` works great on a Dell Latitude 5591! (i5-8400H; `1.1.6` BIOS) It looks like all the expected unlocked settings are available (overclocking menu, SA)
